README.md

```diff
@@ -88,13 +89,71 @@ Additionally, for the development we use:
 
 > See the [Development](#development) section for details on how to install `mockery`.
 
-# Installation
+# Quick-Start Installation
 
-## From binaries
+We provide an installation script to automatically install and update the latest version of Trento.
+
+## Trento Server Installation
+
+T.B.D.
+
+## Trento Agent Installation
+
+You can `curl | bash` if you want to live on the edge.
+
+```
+$ curl -sfL https://raw.githubusercontent.com/trento-project/trento/main/install.sh | sudo bash
+```
+
+Or you can fetch the script, and then execute it manually.
+
+```
+$ curl -O https://raw.githubusercontent.com/trento-project/trento/main/install.sh
+$ chmod 700 install.sh
+$ sudo ./install.sh
+```
+
+The script will ask you for two IP addresses.
+
+- `agent bind IP`: the private address to which the trento-agent should be bound for internal communications.
+  This is an IP address that should be reachable by the other nodes, including the trento server.
+  Note for the Pacemaker users: this IP address _should not be_ a floating IP.
+
+- `trento server IP`: the address where Trento server can be reached.
+
+You can pass these arguments as flags or env variables too:
+
+```
+$ curl -sfL https://raw.githubusercontent.com/trento-project/trento/main/install.sh | sudo bash -s - --agent-bind-ip=192.168.33.10 --server-ip=192.168.33.1
+```
+
+```
+$ AGENT_BIND_IP=192.168.33.10 SERVER_IP=192.168.33.1 sudo ./install.sh
+```
+
+### Start Trento Agent
+
+The installation script does not start the agent automatically.
+
+You can start it by simply:
```

Comment:
In the script `install.sh:182` we say
`echo -e "You can now start trento-agent with: \033[1msystemctl start trento-agent\033[0m"`
We should be consistent either we start _with_, or we start _by_.
Besides, I would put _by_ in the end, like "You can start it simply by"

Reply:
See the other comment, we agreed on "Now you can start the trento-agent with".